packages/solarwinds-apm/package.json

```diff
@@ -45,8 +45,11 @@
     "test": "swtest -p test/tsconfig.json -c src"
   },
   "dependencies": {
+    "@grpc/grpc-js": "^1.9.3",
     "@opentelemetry/auto-instrumentations-node": "^0.39.2",
     "@opentelemetry/core": "~1.17.0",
+    "@opentelemetry/exporter-metrics-otlp-grpc": "~0.43.0",
+    "@opentelemetry/exporter-trace-otlp-grpc": "~0.43.0",
```

Comment:
Follow-on question after seeing @tammy-baylis-swi's comment in https://github.com/solarwindscloud/solarwinds-apm-python/pull/200#pullrequestreview-1638887880, would it work if the exporters were `peerDependencies` instead?

Reply:
Good point, that probably would make more sense

Reply:
Thanks, neat `peers` package to handle this :)